.travis.yml

```diff
@@ -1,6 +1,5 @@
 language: ruby
 script: "script/test_all"
-sudo: false
```

Comment:
Why was this necessary?

Reply:
https://changelog.travis-ci.com/the-container-based-build-environment-is-fully-deprecated-84517

It's a change in Travis CI.